metricbeat/module/kibana/status/status.go

```diff
@@ -72,6 +72,8 @@ func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
 // It returns the event which is then forward to the output. In case of an error, a
 // descriptive error must be returned.
 func (m *MetricSet) Fetch(r mb.ReporterV2) error {
+	apiKey := m.ApiKey
+	m.http.SetHeader("Authorization", "ApiKey "+apiKey)
```

Comment:
[Question]
Same question here: should this always be set or only when `apiKey` is not empty?

Also as a suggestion:
```suggestion
	m.http.SetHeader("Authorization", "ApiKey "+m.ApiKey)
```